cubed/core/array.py

```diff
@@ -105,6 +107,16 @@ def size(self):
         """Number of elements in the array."""
         return reduce(mul, self.shape, 1)
 
+    @property
+    def nbytes(self) -> T_IntOrNaN:
+        """Number of bytes in array"""
+        return self.size * self.dtype.itemsize
```

Comment:
Will this ever be nan? Cubed doesn't have unknown size/shape arrays at the moment.

Reply:
I guess not.